app/controllers/admin/SettingsController.php

```diff
@@ -78,6 +78,18 @@ public function postEdit()
             "default_currency"   => 'required',
 	        "slack_channel"   => 'regex:/(?<!\w)#\w+/',
 	        "slack_botname"   => 'alpha_dash',
+            "ldap_server"   => 'url',
+            "ldap_uname"     => 'min:1',
```

Comment:
You could probably just use the `required` validation rule here, versus `min:1`.